packages/icons/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "lib": ["esnext", "dom"],
+    "target": "es6",
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "jsx": "react",
+    "skipLibCheck": true,
+    "noImplicitAny": true,
+    "baseUrl": ".",
+    "module": "ESNext",
+    "declaration": true,
+    "declarationDir": "dist",
+    "outDir": "dist",
+    "rootDir": ".",
+    "allowSyntheticDefaultImports": true,
+    "plugins": [{ "name": "typescript-plugin-css-modules" }]
+  },
+  "include": [
+    "src/**/*"
+  ],
+  "exclude": [
+    "node_modules",
+    "dist/**/*.js"
+  ]
```

Comment:
we might also want to exclude `__tests__` pattern here

Reply:
Why?

Reply:
I know it's not in the `include`, but anyway `dist` and `node_modules` are excluded, as it is a better practice to be explicit here I believe
and tests usually need a different config
that's the reason the `core`'s config was so overloaded, as it took care for stuff related to the library itself, and storybook, and tests. so we had to, in rollup, do overrides for the publish. it's better to separate concerns

Reply:
I think it can hide errors if you have a different config for tests, and if you want to remove them from dist you can exclude in rollup config

Reply:
We need to inherit the tsconfig of the library's published version and add stuff that are related to tests only in the dedicated config
that's how we stay reliable to the app config but still have specific test-related config in a different one
but usually we need to add stuff which are not related to the lib in the tsconfig to not fail tests